sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -119,9 +119,20 @@ class RadarUser(
     /**
      * The user's current trip.
      */
-    val trip: RadarTrip?
+    val trip: RadarTrip?,
+
+    /**
+     * A boolean indicating whether or not the user's IP address is being mocked, such as in a simulation. May be
+     * `false` if Fraud is not enabled.
+     */
+    val mocked: Boolean = false
```

Comment:
can we also move `proxy` down to group it with these two new properties?

Reply:
No. I specifically did not change the order of variables here, since this is a public-facing API already.